extensions/amp-carousel/0.2/test/test-e2e/helpers.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const TAG_NAME = 'amp-carousel';
+const SCROLLABLE_SLIDE = 'amp-scrollable-carousel-slide';
```

Comment:
I wonder if there is a pattern here we could adopt similar to CSS Modules, allowing the test helper to import the name from the CSS file.

No reason to change for this PR, but in the future it would help to prevent skew between names.

Reply:
Let me see if this is possible.